hi-ml-azure/testazure/testazure/test_azure_util.py

```diff
@@ -821,114 +819,161 @@ def test_set_environment_variables_for_multi_node(
 
     # If all of ENV_MASTER_IP, AZ_BATCHAI_MPI_MASTER_NODE and AZ_BATCH_MASTER_NODE are set, the latter should
     # take precedence. NODE_RANK should get updated to the value of ENV_OMPI_COMM_WORLD_RANK
-    mock_az_batch_master_port = str(randint(1000, MASTER_PORT_DEFAULT - 1))  # Avoid setting to the default value
-    mock_node_rank = str(randint(0, 10))
+    port_mock = str(MASTER_PORT_DEFAULT)  # Avoid setting to the default value
```

Comment:
Good point about not setting to the default value, but the code is no longer doing that:
```suggestion
    port_mock = str(MASTER_PORT_DEFAULT-1)  # Avoid setting to the default value
```